samza-azure/src/main/java/org/apache/samza/system/eventhub/consumer/EventHubSystemConsumer.java

```diff
@@ -219,13 +220,49 @@ private String getNewestEventHubOffset(EventHubClientManager eventHubClientManag
     }
   }
 
+  // Based on the config PerPartitionConnection, create or get EventHubClientManager for the SSP
+  // Note: this should be used only when starting up. After initialization, directly use perPartitionEventHubManagers
+  // to obtain the corresponding EventHubClientManager
+  private EventHubClientManager createOrGetEventHubClientManagerForSSP(String streamId, SystemStreamPartition ssp) {
```

Comment:
Do we have data on what is the overhead in terms or memory? Can you add this to description if you do?